[#5] Write some README (#14)
Holmusk/postgresql-simple-named: README.md

[![Stackage Nightly](http://stackage.org/package/postgresql-simple-named/badge/nightly)](http://stackage.org/nightly/package/postgresql-simple-named)

> [![Build status](https://img.shields.io/travis/Holmusk/postgresql-simple-named.svg?logo=travis)](https://travis-ci.org/Holmusk/postgresql-simple-named)

Reviewer: Should we rearrange badges for consistency?

This library introduces the implementation of named parameters for the
[`postgresql-simple`][pgs] library. `postgresql-simple-named` is considered to
be used along with the [`postgresql-simple`][pgs] library, so you could refer
there for the original documentation of primary functions. This package solves
exclusively one particular problem — gives the ability to use named parameters

> instead of `?` in quasi-quoter queries and offers essential functions for using

Reviewer: Jus to use `use` not so often 🙂 
```suggestion
instead of `?` in quasi-quoter queries and offers essential functions for substituting variables
```
(and delete `them` on next line)

them in queries (`queryNamed`, `executeNamed`).

## Example

Using the provided operator `=?` for named parameters creation and using '?'
prefix for specifying the parameters inside the queries, here is how the query

> could look like:

Reviewer: I would rephrase this like this:

```
Operator `=?` binds named parameters with the corresponding values. 
Named parameters inside SQL query start with the '?' character and can contain lowercase
and uppercase letters, digits and underscore. Below you can find a basic example of how
query with named parameters could look like:
```

```haskell
queryNamed dbConnection [sql|
    SELECT
        id, name, city
    FROM users
    WHERE name = ?nameParam
      AND age  = ?ageParam

|] [ "nameParam" =? "John"
   , "ageParam"  =? 42
   ]
```

> This feature can be extremely helpful when the query using some parameters more than once:

Reviewer: ```suggestion
This feature can be extremely helpful when the query uses some parameters more than once:
```

```haskell
query dbConnection [sql|
    SELECT
        col1, col2
    FROM my_table
    WHERE id = ?
      AND (? IS NULL OR id > ? )
      AND (? IS NULL OR id < ? )

|] (someId, minId, minId, maxId, maxId)
```

This is how the query looks like with the `postgresql-simple` library. You can

> rewrite it the following way:

Reviewer: ```suggestion
rewrite it the following way using the `postgresql-simple-named` library:
```
